hack/update-deps.sh

```diff
@@ -27,15 +27,15 @@ cd ${ROOT_DIR}
 export GO111MODULE=on
 
 # This controls the release branch we track.
-VERSION="master"
+VERSION="release-0.17"
 
 # The list of dependencies that we track at HEAD and periodically
 # float forward in this repository.
 FLOATING_DEPS=(
   "knative.dev/networking@${VERSION}"
   "knative.dev/test-infra@${VERSION}"
   "knative.dev/pkg@${VERSION}"
-  "knative.dev/serving@${VERSION}"
+  "knative.dev/serving@master"
```

Comment:
Is `knative.dev/serving@master` still needed? looks like this repo does not have dependency on `knative.dev/serving`.

Reply:
good point. Removed.